src/Pages/Shop/Option.scss

```
@@ -0,0 +1,75 @@
+@import "../../Styles/common.scss";
+
+.Option {
+  @include flexColumn;
+  position: relative;
+  width: calc(50% - 5px);
+  div {
+    position: relative;
+  }
+
+  .optionNames {
+    z-index: 1;
+    position: absolute;
+    top: 32px;
+    left: 0px;
+    width: 100%;
+    background-color: #fafafa;
+    animation: optionDown 0.1s ease;
+
+    ul {
+      li {
+        display: flex;
+        align-items: center;
+        padding: 10px;
+        font-weight: 400;
+
+        &:hover {
+          font-weight: 600;
+          cursor: pointer;
+        }
+
+        img {
+          width: 30px;
+          height: 10px;
+          padding-right: 10px;
+        }
+      }
+    }
+  }
+  .inActiveOptions {
+    display: none;
+  }
+  .selector {
```

```suggestion
  }
  
  .inActiveOptions {
    display: none;
  }
  
  .selector {
```
구분되는 셀렉터 사이에 한줄 띄워주세요